wrappers/here.rb

```diff
@@ -331,7 +330,9 @@ def get(url_base, object, params = {})
               elsif additional_data.include?({ 'key' => 'error_code', 'value' => 'NGEO_ERROR_ROUTE_NO_START_POINT' })
                 raise UnreachablePointError
               elsif error['subtype'] == 'InvalidInputData'
-                raise RouterWrapper::InvalidArgumentError.new(error), "Here, #{error['subtype']} : #{error['details']} (#{additional_data.first['key']} : #{additional_data.first['value']})"
+                raise RouterWrapper::InvalidArgumentError.new(error), "Here, #{error['subtype']}: #{error['details']} (#{additional_data.first['key']} : #{additional_data.first['value']})"
+              elsif error['subtype'] == 'NoRouteFound'
+                raise RouterWrapper::NoRouteFound.new(error), "Here, #{error['subtype']}: #{params.keys.grep(/waypoint/).map{|key| params[key]}}"
```

Comment:
what is the http status code returned by the API?

Reply:
I changed it to return a 400 like HERE does. It's handled in `api/v01/api.rb`